src/test/resources/k3s/tmforum.yaml

```diff
@@ -131,8 +131,8 @@ spec:
               value: http://scorpio:1026
             - name: GENERAL_BASEPATH
               value: /tmf-api/productCatalogManagement/v4
-          image: quay.io/fiware/tmforum-product-catalog:1.2.7
-          imagePullPolicy: IfNotPresent
+          image: quay.io/wi_stefan/tmforum-product-catalog:1.3.8
+          imagePullPolicy: Always
```

Comment:
Just curious: There is no deeper reason for the different pull policies of the tmf images right?

Reply:
The reason is that I forgot to clean it up;) Will do now